### PR TITLE
Update hangman.py

### PR DESCRIPTION
**Pull request by Soul-Breaker**

### CODE REVIEW

**Comment by srevinsaju** [comment]
Now, add

`HANGMAN_STICKFIGURE` list with each item as different stick mans

here 

https://github.com/sugaroidbot/sugaroid/pull/93/files#diff-10d14580caa78dd34a74b3889f53a1cf4b2ecc7f51892c100a5ed8a4fe2ce27bR284

File: sugaroid/brain/hangman.py [hunk 1]
```diff
@@ -222,6 +222,109 @@
...
+    |   |
+    0   |
+   /|\  |
+   / \  |
+        |
+    ========= 
+"""].split(
+    "\n"
+)[
+    1:-1
```
Comment (srevinsaju) [inline sugaroid/brain/hangman.py:326]: These lines are unnecessary. we cannot split a `list` too. So I guess we can remove the the lines from `].split.... 1:-1`

**Review by srevinsaju — CHANGES_REQUESTED** [review]

File: sugaroid/brain/hangman.py (hunk 1 above)
Comment (Soul-Breaker) [reply sugaroid/brain/hangman.py:326]: ok

**Comment by TheDarkDrake** [comment]
I saw this just not lol. But perfect, @Soul-Breaker corrected some spelling mistakes I see

**Comment by Soul-Breaker** [comment]
> I saw this just not lol. But perfect, @Soul-Breaker corrected some spelling mistakes I see

Well, I realized it after today's trial.

**Comment by srevinsaju** [comment]
Moved to #94

**Pull request closed without merging**